terraform/modules/proxmox/cloudinit/bootloader.sh

```
@@ -39,30 +39,14 @@ function cleanup() {
-  # Clone our playbook and prepare structure of this ansible
-  if [[ ${#repository} -eq 0 ]]; then
-    error "missing repository"
-  fi
+function clone_playbook_from_local_storage() {
+  if ! tar -xzf /tmp/playbook.tar.gz -C /tmp; then
+    error "failed to extract /tmp/playbook.tar.gz to /tmp"
+  fi 
 
-  if ! git clone --branch $branch $repository /tmp/agent; then
-    error "error cloning playbook, please ping SRE to validate network"
+  if ! mv $(find /tmp -name ${repository}) /tmp/playbook; then
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)